Applications/Spreadsheet/JSIntegration.cpp

```diff
@@ -34,6 +34,42 @@
 
 namespace Spreadsheet {
 
+bool can_be_safely_converted(const JS::Value& value)
+{
+    // FIXME: LibJS, please assert less on things you're handing out to your users.
```

Comment:
I disagree with this, at least "from the outside" (JS code rather than LibJS-C++) it should be impossible to pass any of the value types below to a stringification function:

- Empty values should never be leaked, e.g. accessing an array hole returns undefined
- Accessors and native properties can exist in objects, but they always return a primitive or object when accessing them

So what am I missing here?

Reply:
Those kinds of objects likely showed up because the GC was freeing things.
I guess that function has outlived its purpose

Reply:
Hopefully we don't see this kind of GC mishap again :)